app/views/teams/_form.html.erb

```diff
@@ -27,6 +27,17 @@
     </div>
   </div>
 
+  <div class="form-group row">
+          <%= f.label :avatar, t('activerecord.models.image.one'), :class => 'col-form-label col-lg-2'%>
```

Comment:
When I recall correctly, we agreed to not use the rocket Syntax.

Reply:
in unserem code ist die rocket syntax halt 100 mal. Ich änder das jetzt mal in meinem code für den commit, aber wenn wir das generell ändern wollen, müssen wir das fett refactoren.